templates/taxi/driver_form.html

```diff
@@ -0,0 +1,12 @@
+{% extends "base.html" %}
+{% load crispy_forms_filters %}
+
+{% block content %}
+  <h1>{{ object|yesno:"Update,Create" }} driver</h1>

```

Comment:
The use of `object|yesno:"Update,Create"` assumes that `object` is a boolean or can be evaluated as such. Ensure that the context includes an `object` variable that can be correctly evaluated to avoid rendering issues.